app/scripts/modules/amazon/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -34,6 +34,9 @@ module.exports = angular.module('spinnaker.aws.serverGroupCommandBuilder.service
           var keyPair = credentials ? credentials.defaultKeyPair : null;
           var applicationAwsSettings = _.get(application, 'attributes.providerSettings.aws', {});
 
+          var defaultIamRole = settings.providers.aws.defaults.iamRole || '{{application}}InstanceProfile';
+          defaultIamRole = defaultIamRole.replace('{{application}}', application.name);
```

Comment:
Oops ... this comment caused me to realize the default should still be `BaseIAMRole`. I accidentally modified it when doing some local testing and neglected to put it back.